### DISCUSSION TITLE
Stuck on expose client with my own server

### DISCUSSION THREAD

**Discussion by dragonfire1119** [exposedev/expose#230] [Q&A]
I'm trying to set up a docker server. I can access the admin panel & the site but when I use the command on the command line ``expose`` it just hangs. I add -vvv & it shows everything is good with PHP but it just doesn't start up a subdomain with the welcome message or anything. 

```
proxy_pass http://127.0.0.1:8080;
proxy_read_timeout     60;
proxy_connect_timeout  60;
proxy_redirect         off;

proxy_http_version 1.1;
proxy_set_header Upgrade $http_upgrade;
proxy_set_header Connection 'upgrade';
proxy_set_header X-Real-IP $remote_addr;
proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
proxy_set_header X-Forwarded-Proto https;
proxy_set_header Host $host;
proxy_cache_bypass $http_upgrade;
```

**Comment by chrisdicarlo** [comment]
Having similar problems trying to use a reverse proxy.  @dragonfire1119 did you figure out a solution?

I can hit the admin dashboard through the reverse proxy but trying to expose a site just hangs.

**Comment by chrisdicarlo** [comment]
Sorted out my issue - was a configuration problem.

**Comment by dragonfire1119** [comment]
Could you explain how you fixed it? Thanks

**Comment by chrisdicarlo** [comment]
It was a combination of a misconfigured docker container and DNS records being incorrect.

**Comment by dragonfire1119** [comment]
@chrisdicarlo sorry for so many questions but do you happen to know what was misconfigured maybe it could help me in getting this resolved? Thanks for your time!

**Comment by chrisdicarlo** [comment]
I had my container only listening to 127.0.0.1 instead of 0.0.0.0
